server-utils/src/cors.rs

```diff
@@ -103,6 +103,9 @@ impl ops::Deref for Origin {
 	}
 }
 
+// FIXME: There's a potential for allocation optimization here, by moving to `Value(Box<Origin>)`
```

Comment:
IMHO that makes no sense at all. It's again making the public API less ergonomic to make the linter happy